assets/src/dashboard/app/views/editorSettings/publisherLogo/index.js

```diff
@@ -80,30 +83,25 @@ function PublisherLogoSettings({
   const [activePublisherLogo, setActivePublisherLogoId] = useState(null);
   const [indexRemoved, setIndexRemoved] = useState(null);
 
+  const [contextMenuId, setContextMenuId] = useState(null);
+
   const publisherLogosById = useMemo(() => publisherLogos.map(({ id }) => id), [
     publisherLogos,
   ]);
 
+  const hasOnlyOneLogo = publisherLogosById.length === 1;
+
   const publisherLogoCount = useRef(publisherLogosById.length);
 
   const onRemoveLogoClick = useCallback(
-    (e, { publisherLogo, idx }) => {
-      e.preventDefault();
-
+    (publisherLogo, idx) => {
       handleRemoveLogo(publisherLogo);
       setIndexRemoved(idx);
       publisherLogoCount.current = publisherLogosById.length;
     },
     [handleRemoveLogo, publisherLogosById.length]
   );
 
-  // set active logo when first painting
-  useEffect(() => {
-    if (publisherLogos.length > 0 && !activePublisherLogo) {
-      setActivePublisherLogoId(publisherLogos?.[0].id);
-    }
-  }, [activePublisherLogo, publisherLogos, setActivePublisherLogoId]);
-
   // Update publisher logo focus when logo is removed
```

Comment:
We don't want to do this now that there's hidden UI that shows up either when active or hovered upon